hypothesis-python/tests/quality/test_shrink_quality.py

```diff
@@ -343,13 +343,29 @@ def test_lists_forced_near_top(n):
     ) == [0] * (n + 2)
 
 
-def test_sum_of_pair():
+def test_sum_of_pair_int():
     assert minimal(
         tuples(integers(0, 1000), integers(0, 1000)), lambda x: sum(x) > 1000
     ) == (1, 1000)
 
 
-def test_sum_of_pair_separated():
+def test_sum_of_pair_float():
+    assert minimal(
+        tuples(st.floats(0, 1000), st.floats(0, 1000)), lambda x: sum(x) > 1000
+    ) == (1.0, 1000.0)
+
+
+def test_sum_of_pair_mixed():
+    # check both orderings
+    assert minimal(
+        tuples(st.floats(0, 1000), st.integers(0, 1000)), lambda x: sum(x) > 1000
+    ) == (1.0, 1000.0)
+    assert minimal(
+        tuples(st.integers(0, 1000), st.floats(0, 1000)), lambda x: sum(x) > 1000
+    ) == (1.0, 1000.0)
```

Comment:
```suggestion
    # check both orderings
    assert minimal(
        tuples(st.floats(0, 1000), st.integers(0, 1000)), lambda x: sum(x) > 1000
    ) == (1.0, 1000)
    assert minimal(
        tuples(st.integers(0, 1000), st.floats(0, 1000)), lambda x: sum(x) > 1000
    ) == (1, 1000.0)
```
Equal either way, but it's a bit easier to tell what's happening if we compare the expected types.